index.js

```diff
@@ -10,7 +10,8 @@ var noop = function() {}
 module.exports = function(db) {
   var feed = {}
   var lock = mutexify()
- 
+  var valueEncoding = db.options.valueEncoding || 'binary'
```

Comment:
db.options.valueEncoding is always set to my knowledge (defaults to utf8) so that wouldn't work